src/olympia/blocklist/admin.py

```diff
@@ -423,6 +423,7 @@ def add_view(self, request, **kwargs):
             'save_as': False,
             'block_history': self.block_history(
                 self.model(input_guids=guids_data)),
+            'submission_complete': False,
```

Comment:
How is this change related to the issue?

Reply:
it was a drive-by fix for a warning about undefined template variables.

Reply:
I could remove it again.

Reply:
No need to do that. I was just curious.